apps/coscrad-frontend/src/components/image-in-content/image-in-content.tsx

```diff
@@ -0,0 +1,45 @@
+export type ImageAsset = {
+    title: string;
+    src: string;
+    width: number;
+    height: number;
+};
+
+interface ImageInContentProps {
+    image: ImageAsset;
+    displayWidth?: string;
+    title?: string;
+    alt?: string;
+    alignment?: 'left' | 'right';
+}
+
+/**
+ * Could add the caption feature as a part of this.  It could be optional.
+ * The ImageAsset could be the model coming from the digital asset manager in the future
+ * The image could have an optional borderRadius to round the corners
+ * Eventually in the CMS clicking on the image could open it in full screen
+ */
+
+export const ImageInContent = ({ image, alt, displayWidth, alignment }: ImageInContentProps) => {
+    const { title, src, width, height } = image;
+
+    const divStyle = {
```

Comment:
These are my one uses so far of CSS as a const.  Could probably just use `styled`, I can't remember why the exception here except that it might have to do with the `img` tag

Reply:
Interesting. We should analyze the situation and see what we can learn.

Reply:
Thumbs up